lucene/test-framework/src/java/org/apache/lucene/search/AssertingLeafCollector.java

```diff
@@ -50,5 +50,9 @@ public void collect(int doc) throws IOException {
     lastCollected = doc;
   }
 
+  @Override
+  public DocIdSetIterator filterIterator(DocIdSetIterator scorerIterator) {
+    return super.filterIterator(scorerIterator);
```

Comment:
Is this necessary?

Reply:
@romseygeek Good notice, not it is not necessary. Removed in c84fe5e